src/jira/check_jira_refs.rs

```diff
@@ -1,25 +1,36 @@
 use log;
+use conventional::{Commit, Simple as _};
```

Comment:
Random aside this is pretty confusing as a rust noob. Wish traits were imported differently

Reply:
the `Simple as _` thing is confusing for me too.  i also dislike that traits being imported have implicit kinda side effects like this. 😞